src/panels/lovelace/cards/hui-map-card.js

```diff
@@ -11,7 +11,24 @@ import computeStateName from "../../../common/entity/compute_state_name";
 import debounce from "../../../common/util/debounce";
 import parseAspectRatio from "../../../common/util/parse-aspect-ratio";
 
+// should be interface when converted to TS
+export const Config = {
+  title: "",
+  aspect_ratio: "",
+  default_zoom: 14,
+  entities: [],
+};
+
 class HuiMapCard extends PolymerElement {
+  static async getConfigElement() {
+    await import("../editor/config-elements/hui-map-card-editor");
+    return document.createElement("hui-map-card-editor");
+  }
+
+  static getStubConfig() {
+    return { default_zoom: 14, entities: [] };
```

Comment:
Should we remove default_zoom?

Reply:
Yeah, looking back at the docs I believe I misinterpreted them to mean that the default is 14 and not 14/fit

Reply:
So should we remove it?